geth/params/config.go

```diff
@@ -582,7 +582,9 @@ func (c *NodeConfig) updateClusterConfig() error {
 
 	for _, cluster := range clusters {
 		if cluster.NetworkID == int(c.NetworkID) {
-			c.ClusterConfig.BootNodes = cluster.BootNodes
+			if len(c.ClusterConfig.BootNodes) == 0 {
```

Comment:
i am not sure what we really want here

if the goal is testing in isolated network then keeping static nodes doesn't make sense

if the goal is to avoid blockage - then removing our own bootnodes doesn't make sense, right? because how user can be sure for 100% if they are down/laggy or blocked

and i don't see a case when we want to overwrite one but keep another (talking about having static nodes)

Reply:
The goal is to avoid blockage.
The use case is:

Say that status bootnodes IP addresses are blocked in Russia (seems like it might be already the case. We can be quite confident that they are, through user reporting, manual testing etc). 

The idea is to set up a new bootnode and send the IP addresses to the users, through a different channel ( twitter for example).

Would that allow at least for a while the user to use status?

Basically we allow the user to choose bootnodes in advanced network settings.

I have to defer about static-nodes / bootnodes, as I don't know the difference, yet :)

Reply:
static nodes are just normal devp2p (whisper/les) peers, but app tries to connect with them all the time.
boot node serves as entry point to a distributed table of devp2p peers

my main confusion is that we overwrite boot nodes but leave static nodes, but tbh any other variant seems pretty the same. so i nothing against having it as is

Reply:
@dshulyak thanks for the explanation, much clearer.